anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -159,7 +160,7 @@ def post_process(
         if "image_threshold" in meta_data:
             pred_label = pred_score >= meta_data["image_threshold"]
 
-        if task == "segmentation":
+        if task == TaskType.SEGMENTATION:
```

Comment:
Do we have any plans on supporting detection in OpenVINO inference?

Reply:
Inference support will follow. I don't see any problems with adding OpenVINO support, but will need to check to confirm.